resources/readings/Teamwork.md

```diff
@@ -110,7 +110,7 @@ It is extremely important that you remember that your partner is your **best** r
 
 * On [dependence, independence, and interdependence](https://carogiana.wordpress.com/personal-essays/dependence-independence-interdependence-codependence/) from a psychological point of view.
 
-* Another [view](https://classes.soe.ucsc.edu/cmps115/Spring01/staff/teams.html) on SE teamwork.
+* Another [view](https://classes.soe.ucsc.edu/cmps115/Spring01/staff/teams.html) on SE teamwork. 
```

Comment:
Dead link

Reply:
Found a link to replace it with and have updated Teamwork.md.